platform/android/export/export.cpp

```diff
@@ -1455,6 +1483,86 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	Vector<uint8_t> _resize_launcher_icon(const Ref<Image> &p_source_image, const LauncherIcon p_icon) {
```

Comment:
There is a lot of logic that's duplicated from `_process_launcher_icons`, so instead extract the common logic into a separate function.